lib/u3d/unity_runner.rb

```diff
@@ -36,6 +36,7 @@ def run(installation, args, raw_logs: false)
         log_file = installation.default_log_file
       end
 
+      Utils.ensure_dir File.dirname(log_file)
```

Comment:
Is this related to Linux install and running? Or a more general cross platform feature?